src/domain/profile/Profile.tsx

```diff
@@ -48,6 +52,22 @@ const Profile: FunctionComponent = () => {
               <h1>
                 {data.myProfile.firstName} {data.myProfile.lastName}
               </h1>
+              <Button
+                ariaLabel={t('profile.edit.label')}
+                onClick={() => setIsOpen(true)}
+              >
+                <Icon src={settingsIcon} alt={t('profile.edit.label')} />
```

Comment:
Did you miss a span here with edit text?

Reply:
Ah! Good point

Reply:
Added.